tests/test_redact.py

```diff
@@ -32,6 +32,15 @@ def test_redact_svs(svs_input_path, tmp_path, override_rule_set):
     assert b"ICC Profile" not in svs_output_file_bytes
 
 
+def test_redact_svs_image(data_dir, tmp_path):
```

Comment:
Is this test necessary? It's just adding an additional assertion, but operating on effectively the same data as `test_redact_svs`. Could we make both redaction assertions in the existing test?